### ISSUE TITLE
DropDown value issue when ForceSelection set to False

### ISSUE THREAD

**Issue by hajanajubudeen88** [fomantic/Fomantic-UI#805]
# Bug Report

In the TestCase, we have given two Dropdown in that ForceSelection set to False for 1st Dropdown  and  True for 2nd Dropdown. Below is the steps to follow for both the dropdown, in that both dropdown will show different behavior.

## Steps to reproduce
1.  Select any value in both the dropdown and press submit button.
2.  Alert message will display the text of both the dropdown for respective value selected.
3. Now type something which is not present in the dropdown list in both dropdown.
4. On entering different value, First dropdown will hold the incorrect value and second dropdown will change automatically to previous value which was selected from list. First dropdown holds incorrect value because forceselection is set to false.
5. On submitting the dropdown values, first dropdown will throw text of previously selected dropdown value and second dropdown will throw dropdown value which is force selected.

## Expected result

On entering incorrect value in both case, dropdown should hold the incorrect value or empty the value and On submitting the dropdown, both dropdown should return value as null.

## Actual result

On submitting the dropdown after entering values which is not present in dropdown, first dropdown will throw text of previously selected dropdown value and second dropdown will throw dropdown value which is force selected.

## Testcase
https://jsfiddle.net/hajanajubudeen88/qkdLts0n/78/

**Comment by lubber-de** [comment]
I agree, the left dropdown using `forceSelection:false` should provide the value (equal to text in that case).
However, i don't agree the other dropdown (`forceSelection:true`) should provide the manual entered value (and it actually can't because on blur it resets to the previous selected value (which is expected behavior)

**Comment by hajanajubudeen88** [comment]
**Thanks for the Response @lubber-de** 

Is there any solution to fix this issue?
